dotnet/src/Connectors/Connectors.OpenAIV2/Core/Models/AddHeaderRequestPolicy.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+/* Phase 1
+Added from OpenAI v1 with adapted logic to the System.ClientModel abstraction
+*/
+
+using System.ClientModel.Primitives;
+
+namespace Microsoft.SemanticKernel.Connectors.OpenAI;
```

Comment:
Why is this in an OpenAI specific namesapce?

Reply:
Because this is a OpenAI specific Policy, once this is moved as a utility I can remove the namespace and it will use the default from the project.